demos/simple-storage/protocol/recipes/constants.json

```diff
@@ -1,3 +1,3 @@
 {
-  "SimpleStorageAddr": "0x2c5f3c004878923f55A2a255F89Fe29393177509"
+  "SimpleStorageAddr": "0xe78A0F7E598Cc8b0Bb87894B0F60dD2a88d6a8Ab"
```

Comment:
Should this change be included? It may not matter too much since the file is modified with every run.

Reply:
yah doesn't really affect anything :P so it's fine